doc/nrf/libraries/others/emds.rst

```diff
@@ -9,45 +9,47 @@ Emergency data storage
-In general, it should not be necessary to allocate a large flash area, since only a limited set of data should be stored to ensure swift completion of writing the flash on shutdown.
+Allocating a larger flash/RRAM area will demand more resources, but also increase the life expectancy of the flash/RRAM.
+The chosen size should reflect the amount of data stored, the available flash/RRAM resources, and how the application calls the :c:func:`emds_store` function.
+In general, it should not be necessary to allocate a large flash/RRAM area, since only a limited set of data should be stored to ensure swift completion of writing the flash/RRAM on shutdown.
 
 The memory location that is going to be stored must be added on initialization.
 All memory areas must be provided through entries containing an ID, data pointer, and data length, using the :c:func:`emds_entry_add` function and the :c:macro:`EMDS_STATIC_ENTRY_DEFINE` macro.
 Entries to be stored when the emergency data storage is triggered need their own unique IDs that are not changed after a reboot.
 
-When all entries are added, the :c:func:`emds_load` function restores the entries into the memory areas from the flash.
+When all entries are added, the :c:func:`emds_load` function restores the entries into the memory areas from the flash/RRAM.
 
-After restoring the previous data, the application must run the :c:func:`emds_prepare` function to prepare the flash area for receiving new entries.
-If the remaining empty flash area is smaller than the required data size, the flash area will be automatically erased to increase the available flash area.
+After restoring the previous data, the application must run the :c:func:`emds_prepare` function to prepare the flash/RRAM area for receiving new entries.
+If the remaining empty flash/RRAM area is smaller than the required data size, the flash/RRAM area will be automatically erased to increase the available flash/RRAM area.
 
 The storage is done in deterministic time, so it is possible to know how long it takes to store all registered entries.
 However, this is chip-dependent, so it is important to measure the time.
 Find timing values under the "Electrical specification" section for the non-volatile memory controller in the Product Specification for the relevant SoC or the SiP you are using.
-For example, for the nRF9160 SiP, see the `Electrical specification of nRF9160`_ page.
+For example, for the nRF52840 SiP, see the `nRF52840 Product Specification`_ page.
 
 The following Kconfig options can be configured:
 
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US`
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`
 
-When configuring these values, consider the time for erase when doing garbage collection in NVS.
+When configuring these values, consider the time for erase when doing garbage collection in NVS using flash.
```

Comment:
NVS is doing garbage collection both with flash and RRAM isn't it? Should this line stay as it was?

Reply:
I would not add ZMS, as I do not think it is recommended to use ZMS together with flash. That is way I said using NVS together with flash.